src/GraphQL.Tests/Builders/ConnectionBuilderTests.cs

```diff
@@ -394,6 +394,28 @@ public void bidirectional_called_twice_creates_proper_arguments()
             graph.Fields.Find("connection").Arguments.Count(x => x.Name == "last").ShouldBe(1);
         }
 
+        [Fact]
+        public void should_use_pagesize()
+        {
+            var graph = new ObjectGraphType();
+            graph.Connection<ChildType>()
+                .Name("connection")
+                .PageSize(10)
+                .Resolve(context => context.First);
```

Comment:
What if you swap these lines?

Reply:
Setting `PageSize` and other property after calling `Resolve` now does nothing.

Reply:
This is not possible in the connection builders included in 4.3 because Resolve returns void.  That's probably why they returned void.

The typed connection builders return the builder, but they have not been released yet.  I suggest altering them to return void.

I really don't like the fact that every time a connection is resolved it evaluates both these lines:

```cs
FieldType.GetMetadata<int?>(ConnectionBuilder<TSourceType>.PAGE_SIZE_METADATA_KEY);

FieldType.Arguments.Find("before")?.Type == typeof(StringGraphType) && FieldType.Arguments.Find("last")?.Type == typeof(IntGraphType);
```

which is a dictionary lookup and two list enumerations every time that the resolver executes.  This did not occur in 4.2.

Reply:
OK, then make it `void` and put some notes into docs/xml.

Reply:
Although this approach looks strange and will cause other questions.

Reply:
I was wrong; the typed connection builders are included in 4.3 and do return the connection builder.  It is the scoped connection builders that did not make it into 4.3.  The untyped connection builders from 4.2 (and also in 4.3) return void.

In this case I would make a breaking change and change the typed connection builders to return void.  4.3 is only 5 days old.  We just need to note the change in the release notes.

I have made the changes as described and are ready for your review.